packages/eventcatalog-types/src/index.d.ts

```diff
@@ -61,6 +61,13 @@ export interface Link {
   href: string;
 }
 
+export interface openGraphConfig {
```

Comment:
Cool most of this looks great, I would uppercase the word here just to follow convension 

```suggestion
export interface OpenGraphConfig {
```

Reply:
Fixed